### ISSUE TITLE
Learning to learn by gradient descent by gradient descent

### ISSUE THREAD

**Issue by icoxfog417** [arXivTimes/arXivTimes#94]
## 一言でいうと

ニューラルネットの学習ってなんだかんだ手作業orルールベースだよね、ということで、「学習のさせ方」のノウハウを学習するという研究。ターゲットのネットワークの重みを、RNN(2層LSTM)を使って推定させる。AdamやRMSpropより優秀な結果。TensorFlow実装有り。

### 論文リンク

https://arxiv.org/abs/1606.04474

### 著者/所属機関

Marcin Andrychowicz, Misha Denil, Sergio Gomez, Matthew W. Hoffman, David Pfau, Tom Schaul, Brendan Shillingford, Nando de Freitas

* Google DeepMind
* University of Oxford
* Canadian Institute for Advanced Research

**Comment by icoxfog417** [comment]
TensorFlowでの実装が公開された

https://github.com/deepmind/learning-to-learn
